Source/Core/Common/IniFile.cpp

```diff
@@ -292,17 +292,29 @@ bool IniFile::Load(const std::string& filename, bool keep_current_data)
           std::string key, value;
           ParseLine(line, &key, &value);
 
-          // Lines starting with '$', '*' or '+' are kept verbatim.
-          // Kind of a hack, but the support for raw lines inside an
-          // INI is a hack anyway.
-          if ((key.empty() && value.empty()) ||
-              (!line.empty() && (line[0] == '$' || line[0] == '+' || line[0] == '*')))
+          // hack to support the concept of the gameconfig.txt inside of the ini
+          if (current_section->name == "Gecko_GameConfig")
           {
             current_section->m_lines.emplace_back(line);
+            if (!key.empty() || !value.empty())
+            {
+              current_section->Set(key, value);
```

Comment:
Not sure what this accomplishes here: I would assume `[Gecko_GameConfig]` should not have any key-value pairs?

Reply:
there is a couple items that gecko's gameconfig.txt that can be interpreted as key value pairs, this is just a convenience so that I don't have to manually parse those.